pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildPexTask.java

```diff
@@ -16,13 +16,13 @@
 package com.linkedin.gradle.python.tasks;
 
 import com.linkedin.gradle.python.extension.DeployableExtension;
-import com.linkedin.gradle.python.extension.PexExtension;
+import com.linkedin.gradle.python.util.PexApplication;
```

Comment:
This is used as an extension later. Should this class also be in the `extension` package, rather than in the `util` package?

Reply:
I (probably mis-)remember talking about this at a sprint.  I think it was because the class implements `ApplicationContainer`, but on further reflection, you might be right that it's better named as an extension.  I don't mind reverting this back, though I suppose I should also rename it back to `PexExtension`.  Let me try that and see how it looks.

Reply:
Okay, that looks fine, so I've gone ahead and reverted that back to `PexExtension`.